tests/cucumber/features/channels/channels_home.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Home channels
+  As a public user
+  I want to browse all channels
+
+  Background:
+    Given I am on the home page
+
+  # Test channels visible on the home page
+  @dev
+  Scenario: Can't see any channels when DB is empty
+    Given the database has "0" channels
+    Then I should see "0" channels
+
+  @dev
+  Scenario: Can see 5 channels on the home page
```

Comment:
This scenario title is out of date. It could be more generic and say something like "List latest channels"